iron-selectable.html

```diff
@@ -310,11 +310,6 @@
     },
 
     _activateHandler: function(e) {
-      // TODO: remove this when https://github.com/Polymer/polymer/issues/1639 is fixed so we
```

Comment:
I'm closing this PR in favor of https://github.com/PolymerElements/iron-selector/pull/36